packages/react-composites/src/composites/CallComposite/components/HoldPane.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PrimaryButton, Stack, Text, useTheme } from '@fluentui/react';
+import { _pxToRem } from '@internal/acs-ui-common';
+import React, { useState } from 'react';
+import { useLocale } from '../../localization';
+import {
+  holdPaneContentStyles,
+  holdPaneLabelStyles,
+  holdPaneTimerStyles,
+  paneStyles,
+  resumeButtonStyles
+} from '../styles/HoldPane.styles';
+
+/**
+ *
+ */
+export interface HoldPaneProps {
+  onToggleHold: () => Promise<void>;
+}
+
+/**
+ * Hold pane to display when the user places themselves on hold
+ *
+ * @beta
+ */
+export const HoldPane = (props: HoldPaneProps): JSX.Element => {
+  const { onToggleHold } = props;
+  const locale = useLocale();
+  const theme = useTheme();
+
+  const [rawTime, setRawTime] = useState<number>(0);
+  const [time, setTime] = useState<string>('00:00');
```

Comment:
we don't need two states here, one for time as a number and one for time as a string, simply store the duration number in state and for the string you can do that calculation in the regular body:
```
const [timeElapsed, setTimeElapsed] = useState<number>(0);
const elapsedTimeString = `${getMinutes(timeElapsed)}:${getSeconds(timeElapsed)}`;
```